tests/conftest.py

```diff
@@ -0,0 +1,80 @@
+"""Parametrization of validation tests.
+
+Each namespace will form a single suite, with each of the tests either passing
+or failing validation based on 
+http://doc.pytest.org/en/latest/example/parametrize.html
+"""
+
+import pytest
+from pathlib import Path
+import json
+
+ROOT = Path(__file__).parent.parent
+SCHEMAS_ROOT = ROOT / "schemas"
+VALIDATION_ROOT = ROOT / "validation"
+
+
+@pytest.fixture()
+def schemas():
+    schemas = {}
+    for path in SCHEMAS_ROOT.glob("**/*.schema.json"):
+        assert (
+            len(path.relative_to(ROOT).parts) <= 4
+        ), f"schemas directory too deep: {path}"
+        # Using the path parts allows proper handling of schemas that are not
+        # placed in the expected directory structure, like pioneer-study.
+        #
+        #   >>> schema.parts
+        #   ('validation', 'pocket', 'fire-tv-events.1.sample.pass.json')
+        #
+        # https://docs.python.org/3/library/pathlib.html#accessing-individual-parts
+        namespace = path.relative_to(ROOT).parts[1]
+        doctype, version = path.name.split(".")[:2]
+        qualifier = f"{namespace}.{doctype}.{version}"
+        with path.open() as fp:
+            schemas[qualifier] = json.load(fp)
+    return schemas
+
+
+# TODO: test the following cases
+# - example in wrong place
+# - example with the wrong name
+# - schema that doesn't exist
+def pytest_generate_tests(metafunc):
```

Comment:
One interesting idea here is to generate tests to make sure that every schema has an example, but that's for another PR.